src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -125,7 +125,9 @@ private void setUpEventHandler() {
     private Node createFilterBox() {
         filterTextField = new FilterTextField("")
                 .setOnConfirm((text) -> {
+                    Platform.runLater(() -> ui.triggerEvent(new PanelReloadingEvent()));
                     applyStringFilter(text);
+                    Platform.runLater(() -> ui.triggerEvent(new PanelReloadedEvent()));
```

Comment:
Does this take in to account if some of the `repo`s are changed in the filter? Since opening a repo happens asynchronously with `applyStringFilter(text)` and takes quite some time, so the loading indicator is most useful for it.